src/_addCorruptFileErrorToReport.ts

```diff
@@ -0,0 +1,33 @@
+import _setWith from "lodash/setWith";
+import { ErrorReport } from "./migration.types";
+
+/**
+ * Reports that no entry matching `fileId` was found in the `/ui/${contentType}/structure` folder.
+ */
+export function _addCorruptFileToReport(
```

Comment:
```suggestion
export function _addCorruptFileErrorToReport(
```
Just a suggestion, to be more in line with the name of `_addErrorToReport`. Also it's more accurate, we're adding the error itself to the report.